src/EnergyPlus/UnitarySystem.cc

```diff
@@ -2696,6 +2696,47 @@ namespace UnitarySystems {
+                                                                0.0,
+                                                                this->m_MaxONOFFCyclesperHour,
+                                                                this->m_HPTimeConstant,
+                                                                this->m_FanDelayTime,
+                                                                DataHVACGlobals::CompressorOperation::Off,
+                                                                0.0,
+                                                                FirstHVACIteration);
+                state.dataSize->DataConstantUsedForSizing = WaterToAirHeatPumpSimple::GetCoilCapacity(
+                    state, DataHVACGlobals::cAllCoilTypes(this->m_HeatingCoilType_Num), this->m_HeatingCoilName, ErrFound);
+                EqSizing.DesHeatingLoad = state.dataSize->DataConstantUsedForSizing;
+                state.dataSize->DataFractionUsedForSizing = 1.0;
+                SizingMethod = DataHVACGlobals::AutoCalculateSizing;
+                this->m_DesignHeatingCapacity = DataSizing::AutoSize;
+                if (this->m_CoolingCoilType_Num == DataHVACGlobals::Coil_CoolingWaterToAirHPSimple)
+                    // adjusted cooling coil capacity
+                    WaterToAirHeatPumpSimple::SimWatertoAirHPSimple(state,
+                                                                    blankString,
+                                                                    this->m_CoolingCoilIndex,
+                                                                    this->m_CoolingCoilSensDemand,
+                                                                    this->m_CoolingCoilLatentDemand,
+                                                                    0.0,
+                                                                    0.0,
+                                                                    this->m_MaxONOFFCyclesperHour,
+                                                                    this->m_HPTimeConstant,
+                                                                    this->m_FanDelayTime,
+                                                                    DataHVACGlobals::CompressorOperation::Off,
+                                                                    0.0,
+                                                                    FirstHVACIteration);
+                state.dataSize->DataConstantUsedForSizing = WaterToAirHeatPumpSimple::GetCoilCapacity(
+                    state, DataHVACGlobals::cAllCoilTypes(this->m_CoolingCoilType_Num), this->m_CoolingCoilName, ErrFound);
+                EqSizing.DesCoolingLoad = state.dataSize->DataConstantUsedForSizing;
+                state.dataSize->DataFractionUsedForSizing = 1.0;
+                SizingMethod = DataHVACGlobals::AutoCalculateSizing;
+                this->m_DesignCoolingCapacity = DataSizing::AutoSize;
             }
```

Comment:
There is really no good reason to call a child component inside UnitarySystem sizeSystem. What is the difference between the cooling capacity found in step 2 and the return from this call?

Reply:
One reason (not necessarily a good reason) is that we need/want to report the resulting size of every autosized field. When the parent object reports capacities, then you end up with some duplicate numbers. But UnitarySystem does not have capacity fields, so why is it calculating and reporting capacities and not just calling the coils? Well, because the coil sizing needs to coordinate between heating/cooling/supp heat, esp for heat pumps.